corehq/apps/app_manager/views/forms.py

```diff
@@ -162,7 +164,9 @@ def edit_advanced_form_actions(request, domain, app_id, module_id, form_id):
 def edit_form_actions(request, domain, app_id, module_id, form_id):
     app = get_app(domain, app_id)
     form = app.get_module(module_id).get_form(form_id)
+    old_form_case_load = form.form_case_load
```

Comment:
Looks like this is not always set. See [FB 220675](http://manage.dimagi.com/default.asp?220675)